### ISSUE TITLE
[Accuracy] convnextv2_nano.fcmae_ft_in22k_in1k fails bfloat16 training on XPU (pass -> fail_accuracy)

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#5499]
### Summary

The timm_models model **`convnextv2_nano.fcmae_ft_in22k_in1k`** regresses from `pass` to `fail_accuracy` on XPU for **bfloat16 / training** in the Inductor E2E accuracy run.

### Details

| Suite | Model | Dtype | Mode | Previous | Current |
|---|---|---|---|---|---|
| timm_models | convnextv2_nano.fcmae_ft_in22k_in1k | bfloat16 | training | pass | fail_accuracy |

- Batch size: 8
- Calls captured: 289, unique graphs: 1, graph breaks: 5
- All other dtypes (float32, float16, amp_bf16, amp_fp16) for this model still `pass` in training.

### Environment

- Commit: `6a2d048e453e0d63324c34a98dfeaf6921f433eb`
- Run: Inductor wheel on-demand LTS2 XPU E2E (run id `35682598578`)

### Reproduce

```bash
cd pytorch
python benchmarks/dynamo/timm_models.py \
  --accuracy --training --bfloat16 \
  --backend=inductor --device=xpu \
  --only convnextv2_nano.fcmae_ft_in22k_in1k
```

### Detailed log

Failure output:

```
xpu  train convnextv2_nano.fcmae_ft_in22k_in1k
E0922 03:46:41.412000 74178 torch/_dynamo/utils.py:3821] RMSE (res-fp64): 0.00284, (ref-fp64): 0.00033 and shape=torch.Size([80]). res.dtype: torch.bfloat16, multiplier: 3.000000, tol: 0.010000, use_larger_multiplier_for_smaller_tensor: 0
E0922 03:46:41.412000 74178 torch/_dynamo/utils.py:3653] Accuracy failed for key name stages.0.blocks.0.conv_dw.bias.grad
fail_accuracy
```

Accuracy fails on the gradient of `stages.0.blocks.0.conv_dw.bias`: RMSE(res-fp64)=0.00284 vs RMSE(ref-fp64)=0.00033 (shape `[80]`), exceeding the bf16 tolerance (multiplier 3.0, tol 0.01).

**Comment by msnliu** [comment]
passed on b70:
<img width="1976" height="115" alt="Image" src="https://github.com/user-attachments/assets/01b9307d-8d64-4faf-b7f2-73c5a89bc67a" />
will check pvc and update tomorrow

**Comment by kaileiyx** [comment]
pytorch 2.14 pre-release passed on PVC 1550
```bash
(myvenv) root@4030646d5063:/workspace/pytorch# python benchmarks/dynamo/timm_models.py   --accuracy --training --bfloat16   --backend=inductor --device=xpu   --only convnextv2_nano.fcmae_ft_in22k_in1k
loading model: 0it [00:00, ?it/s]Warning: You are sending unauthenticated requests to the HF Hub. Please set a HF_TOKEN to enable higher rate limits and faster downloads.
WARNING:huggingface_hub.utils._http:Warning: You are sending unauthenticated requests to the HF Hub. Please set a HF_TOKEN to enable higher rate limits and faster downloads.
loading model: 0it [00:03, ?it/s]
xpu  train convnextv2_nano.fcmae_ft_in22k_in1k
W0924 09:02:37.733000 1531 torch/_inductor/cudagraph_utils.py:401] [2/0_1] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
pass
(myvenv) root@4030646d5063:/workspace/pytorch# pip list | grep torch
torch                  2.14.0.dev20260804+xpu
torchaudio             2.11.0.dev20260805+xpu
torchvision            0.29.0.dev20260805+xpu
(myvenv) root@4030646d5063:/workspace/pytorch# git branch
* (HEAD detached at 0163eec5ea4)
  main
```

**Comment by msnliu** [comment]
pytorch 2.15 passed with torch-xpu-ops at Commit: 6a2d048e453e0d63324c34a98dfeaf6921f433eb on PVC
```bash
(torch-2.15-dev) gta@DUT1024PVC:~/msnliu/pytorch-2.15$ python ~/msnliu/pytorch-2.15/benchmarks/dynamo/timm_models.py   --accuracy   --training   --bfloat16   --backend=inductor   --device=xpu   --only=convnextv2_nano.fcmae_ft_in22k_in1k
loading model: 0it [00:00, ?it/s]Warning: You are sending unauthenticated requests to the HF Hub. Please set a HF_TOKEN to enable higher rate limits and faster downloads.
WARNING:huggingface_hub.utils._http:Warning: You are sending unauthenticated requests to the HF Hub. Please set a HF_TOKEN to enable higher rate limits and faster downloads.
loading model: 0it [00:03, ?it/s]
xpu  train convnextv2_nano.fcmae_ft_in22k_in1k 
W0924 01:30:42.956000 2603694 torch/_inductor/cudagraph_utils.py:405] [2/0_1] [__cudagraphs] skipping cudagraphs due to skipping cudagraphs due to multiple devices: device(type='xpu', index=0)
pass
(torch-2.15-dev) gta@DUT1024PVC:~/msnliu/pytorch-2.15$ pip list | grep torch
torch              2.15.0a0+git82e11ef /home/gta/msnliu/pytorch-2.15
torchvision        0.29.0+xpu
(torch-2.15-dev) gta@DUT1024PVC:~/msnliu/pytorch-2.15$ git branch
* (HEAD detached at 82e11efd)
  main
```

**Comment by mengfei25** [comment]
@msnliu Thanks, checked in latest nightly https://github.com/intel/torch-xpu-ops/actions/runs/35866670045, and PASS
